src/main/java/it/gov/pagopa/payhub/activities/activity/debtposition/AuthorizeOperatorOnDebtPositionTypeActivity.java

```diff
@@ -0,0 +1,54 @@
+package it.gov.pagopa.payhub.activities.activity.debtposition;
+
+import it.gov.pagopa.payhub.activities.dao.DebtPositionTypeOrgDao;
+import it.gov.pagopa.payhub.activities.dto.DebtPositionTypeOrgDTO;
+import it.gov.pagopa.payhub.activities.exception.custom.OperatorNotAuthorizedException;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+/**
+ * Service class responsible for verifying authorization on the DebtPositionType entity.
+ * This class provides methods to verify whether an operator has authorization to manage installment types
+ * associated with a particular organization.
+ */
+
+@Service
+public class AuthorizeOperatorOnDebtPositionTypeActivity {
+
+    private final DebtPositionTypeOrgDao debtPositionTypeOrgDao;
+
+    /**
+     * Constructs a new {@code VerifyAuthorizationInstallmentsActivity} instance.
+     *
+     * @param debtPositionTypeOrgDao the data access object for {@code DebtPositionTypeOrg} entities
+     */
+
+    public AuthorizeOperatorOnDebtPositionTypeActivity(DebtPositionTypeOrgDao debtPositionTypeOrgDao) {
+        this.debtPositionTypeOrgDao = debtPositionTypeOrgDao;
+    }
+
+    /**
+     * Verifies if the specified operator has authorization to manage a specific installment type for an organization.
+     * <p>
+     * This method retrieves an authorized {@link DebtPositionTypeOrgDTO} associated with the specified organization
+     * and checks if the operator is authorized to manage it based on the provided {@code orgId},
+     * {@code debtPositionTypeOrgId}, and {@code username}.
+     * </p>
+     *
+     * @param orgId the identifier of the organization
+     * @param debtPositionTypeOrgId the identifier of the specific debt position type to verify for authorization
+     * @param username the username of the operator
+     * @return the {@link DebtPositionTypeOrgDTO} representing the authorized installment type
```

Comment:
the type is of the entire debtPosition, not the particular installment
```suggestion
     * @return the {@link DebtPositionTypeOrgDTO} representing the authorized debit position type
```